pcweb/pages/faq.py

```diff
@@ -0,0 +1,135 @@
+
+from pcweb.styles import text_colors as tc
+from pcweb.styles import colors as c
+
+faq_items = [
+    {
+        "Q": "What is Reflex?",
+        "A": rx.text(
+            "Reflex is an open-source, full-stack python framework that makes it easy to build and deploy web apps in minutes. It offers the ease of use and accessibility of low-code frameworks, combined with the flexibility, performance, and customizability of traditional web development. Reflex is open-source and designed to be easy to get started with for those with no previous web development experience."
+        ),
+    },
+    {
+        "Q": "What can I build with Reflex?",
+        "A": rx.vstack(
+            rx.text(
+            """
+            With Reflex, data scientists and software engineers can create high-quality web applications quickly and easily without needing to learn specific web development technologies. Whether you want to build a small-scale data science project/internal app, or a large-scale multi-page web app, Reflex has the tools and features to handle any project. 
+            """),
+            rx.text(
+                "Check out our ",
+                rx.span(doclink("gallery", href="/docs/gallery")),
+                " to see what you can build with Reflex.",
+            ),
+            align_items="flex-start",
+            width="100%",
+        )
+    },
+    {
+        "Q": "Whats the status on hosting?",
+        "A": rx.text(
+            """
+            Our hosting service is being actively developed and is aimed to be released Fall 2023. In the meantime, you can deploy your Reflex app to any hosting service following our 
+            """,
+            rx.span(doclink("self-hosting guide", href="/docs/hosting/self-hosting")),
+            ".",
+        ),
+    },
+    {
+        "Q": "How can I contribute?",
+        "A": rx.text(
+            """
+            We're always looking for contributors to help us build Reflex. 
+            If you're interested in contributing, check out our 
+            """,
+            rx.span(doclink("contributor program", href=constants.CONTRIBUTOR_PROGRAM_URL)),
+            "."
+        ),
+    },
+    {
+        "Q": "How does this compare existing solutions?",
+        "A": rx.vstack(
+            rx.text("""
+            Web apps are the most common way for developers to share their ideas. But even for skilled engineers, without experience in traditional frontend tools like Javascript or React, making a web app can be overwhelming and time-consuming. And once their app is created, deploying it is often a nightmare. Over the years, many low-code and no-code frameworks have tried to make web development more accessible, but they all have limitations and graduation risks compared to standard web frameworks. With Reflex, we have created a framework that lets developers leverage their existing Python skills to build and deploy apps without compromising on flexibility or customization.
```

Comment:
not sure exactly what, but i think this question needs a sentence or paragraph about how nice it is to not have to write a data layer and API layer to get content to your frontend, which can be a big slowdown, especially if backend is not javascript

Reply:
These comments are great! We can add a follow up for this point I added the rest.